devimint/src/main.rs

```diff
@@ -974,8 +977,9 @@ async fn main() -> Result<()> {
             task_group.make_handle().make_shutdown_rx().await.await?;
         }
         Cmd::RunUi(kind) => {
+            let fed_size = args.common.fed_size;
             let (process_mgr, task_group) = setup(args.common).await?;
-            run_ui(&process_mgr, &task_group, &kind).await?
+            run_ui(&process_mgr, &task_group, &kind, fed_size).await?
```

Comment:
not blocking, 
but you can also use `process_mgr.globals.FM_FED_SIZE` this is set in `setup` function `var::Global::new(..., 4)`

Reply:
Got it. New to the Rust side so appreciate the pointers. I just looked for somewhere else where we were accessing an env var and tried to copy the pattern.

Down in globals, it looks to me like it's hard coded to 4? Is there something I'm missing where it's reading in the env var? Or are you recommending I update that `4` to be `arg.fed_size` instead?

Reply:
yes, that 4 should change to `arg.fed_size`

Reply:
you can also try replacing other 4s with globals.FM_FED_SIZE  (`rg '\b4\b' devimint/src`)
But can also be done in a followup.

Reply:
Went ahead and did it here. Thanks for the note!